src/Uno.Sdk/targets/Uno.DefaultItems.targets

```diff
@@ -2,6 +2,12 @@
 	<ItemGroup>
 		<UpToDateCheckInput Include="**\*.xaml" Exclude="$(DefaultItemExcludes)" IsDefaultItem="true" />
 	</ItemGroup>
+	
+	<PropertyGroup>
+		<!-- It's never useful to include bin/obj folders, even when UseArtifactsOutput is false  -->
+		<DefaultItemExcludes Condition="!$(DefaultItemExcludes.Contains('bin/**'))">$(DefaultItemExcludes);bin/**</DefaultItemExcludes>
```

Comment:
I thought these should have been included from the .NET SDK itself?

Reply:
It is, but only when `UseArtifactsOutput` is set. Otherwise, it's using `bin\$(Configuration)` and `obj\$(Configuration)` which includes the other configuration if built previously...